### DISCUSSION TITLE
在 Obsidian 当中如何嵌套多个链接？

### DISCUSSION THREAD

**Discussion by Konjac-XZ** [oldwinter/knowledge-garden#82] [提问]
Hello，我是一名 Obsidian 的新用户。在少数派拜读了您的《Obsidian 是可以用一辈子不换的笔记软件吗？》之后有一点困惑，花了很长的时间 Google / Bing 都没有找到答案（可能是我搜索的姿势不对？），所以冒昧来问问您。
![image](https://github.com/oldwinter/knowledge-garden/assets/71483384/0d839c03-2177-43c4-a948-22ee36403db3)
这段话里，您说给一句话创立链接之后，再分别给三个词语创建链接，但是我遇到的效果是这样的：
![image](https://github.com/oldwinter/knowledge-garden/assets/71483384/edbfcafb-c47c-4996-ae67-283a4983327f)
……所以您的意思是把这句话再复制一遍？像这样？
![image](https://github.com/oldwinter/knowledge-garden/assets/71483384/617784eb-7350-402b-92b8-ddbc09e1e105)

我知道这么提问对于写过代码的人而言显得很蠢，但我实在是被这个问题憋屈得很，还请您赐教。

**Comment by oldwinter** [comment]
哈哈，不好意思，可能是我表达上误导了你。你想要的操作obsidian是不支持的。我文中想表达的效果可以用2个方法实现：

## 方法1
- [[生活中只有一种英雄主义，那就是再认清生活真相后依然热爱生活。]]
  - [[英雄主义]]
  - [[真想]]

后续看到[[英雄主义]]这篇笔记的时候，反向链接面板里能看到完整句子的上下文信息。

## 方法2
在[[生活中。。]]文档里面，直接写入[[英雄主义]]，[[真相]]等文本。

这种用法像标签。


殊途同归，根本目的都是通过比如查看[[英雄主义]]这篇笔记的时候，能看到曾经在[[生活中。。]]这篇笔记里面，被引用过。
